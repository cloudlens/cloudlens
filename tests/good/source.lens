stream {
    print(e.message)
}

source ("local:tests/good/source.log2")

stream {
    print(e.message)
}
