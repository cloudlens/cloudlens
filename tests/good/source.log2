bar 1
bar 2
bar 3
bar 4
bar 5
bar 6