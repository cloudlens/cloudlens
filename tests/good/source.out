foo 1
foo 2
foo 3
foo 4
bar 1
bar 2
bar 3
bar 4
bar 5
bar 6
